src/main/resources/default/extensions/wondergem-page/storage-vfs-modals.html.pasta

```diff
@@ -105,7 +105,7 @@
         </w:modal>
 
         <script type="text/javascript">
-            function selectVFSFile(initialPath) {
+            function selectVFSFile(initialPath, restictToPath) {
```

Comment:
filefield.html.pasta uses it at well.
Code using restrictToPath still in progress or not yet pushed?